platform/ios/src/MGLMapView.mm

```diff
@@ -122,6 +122,9 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingState) {
 
 const CGSize MGLAnnotationAccessibilityElementMinimumSize = CGSizeMake(10, 10);
 
+// Context for KVO observing UILayoutGuides.
+static char MGLLayoutGuidesUpdatedContext = 1;
```

Comment:
Er, I may be mistaken about `_cmd`. But in any case, the important thing is to ensure that the context’s _value_ is unique. If we later add a key-value observer with a different constant set to 1, suddenly there’s an opportunity for crosstalk.

A simple fix is to [set the context equal to its own address](http://nshipster.com/key-value-observing/#correct-context-declarations).